admin/app/views/switchboard.scala.html

```diff
@@ -1,44 +1,54 @@
 @import conf.switches.Switch
 @(env: String, lastModified: Long)(implicit flash: Flash, request: RequestHeader)
-
 @import controllers.admin.routes.UncachedAssets
+@defining(conf.switches.Switches.grouped){ switchGroups =>
+    @admin_main("Switchboard", env, isAuthed = true) {
+    <link href="@UncachedAssets.at("css/switchboard.css")" rel="stylesheet">
+    <link href="@UncachedAssets.at("css/radiator.css")" rel="stylesheet">
+    @if(flash.get("error").isDefined) { <h1 style="color:#bd362f">@flash.get("error").get</h1> }
+    <p>
+        <blockquote>
+            <em>"Death by switches."</em>
+            <footer>Nick Haley</footer>
+        </blockquote>
+    </p>
+    <div>
+        <h3>Switch Groups</h3>
+        <ul class="multi-column--3">
+        @for((group, _) <- switchGroups){
+            <li><a href="#@group.name">@group.name</a></li>
```

Comment:
Is there a Scala thing that automatically encode string so that they're URL-safe?